modules/installation-configuration-parameters.adoc

```diff
@@ -1134,14 +1162,30 @@ The `licenses` parameter is a deprecated field and nested virtualization is enab
 |For control plane machines, the ID of the project in which the KMS key ring exists. This value defaults to the VM project ID if not set.
 |The GCP project ID.
 
-////
-`controlPlane.platform.gcp.osDisk.encryptionKey.kmsKeyServiceAccount`
+|`controlPlane.platform.gcp.osDisk.encryptionKey.kmsKeyServiceAccount`
+|The GCP service account used for the encryption request for control plane machines. If absent, the Compute Engine default service account is used. For more information on GCP service accounts, see Google's documentation on link:https://cloud.google.com/compute/docs/access/service-accounts#compute_engine_service_account[service accounts].
```

Comment:
"on" or "about" service accounts?

(reflects use throughout PR.)

Reply:
Switched them all over to "about", thanks